packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/ManifestHeaderData.kt

```diff
@@ -0,0 +1,8 @@
+package expo.modules.updates.manifest
+
+data class ManifestHeaderData(
+  val protocolVersion: String?,
+  val serverDefinedHeaders: String?,
+  val manifestFilters: String?,
+  val manifestSignature: String?
```

Comment:
If data classes support this, I think it would make the callers more legible if these parameters had default values of `= null` and the callers used named arguments. The tests would then look like `ManifestHeaderData(protocolVersion = "0")`